docs/faq.rst

```diff
@@ -712,18 +712,28 @@ will make the subsequent rebuilds faster (because it needs to transfer less data
 from the repository). While being faster, the cache needs quite some disk space,
 which might be unwanted.
 
-There is a temporary (but maybe long lived) hack to avoid using lots of disk
-space for chunks.archive.d (see :issue:`235` for details):
+A temporary (but maybe long lived) hack to avoid using lots of disk memory
+for ``chunks.archive.d`` (see :issue:`235` for details) is to replace this
+directory with an empty dummy file of this name.  Note, below,
+``$(borg config id)`` is used as place holder for the alphanumeric string
+specific to your backup repository (REPO_ID) you have to enter instead.
```

Comment:
This is not a placeholder that has to get manually replaced.

The shell will call the command `borg config id` and replace that term by the output of that command.

The command in this short form will only work if `BORG_REPO` env var is set, otherwise one would have to add the repo location to the command.

Reply:
Also, because `borg config` only works for local repos, this does not work for remote repos.

Reply:
So far, I used borg only indirectly with vorta as a GUI.  Based on your explanation I recognize I did not comprehend the intent of ``$(borg config id)``, and retract the suggestion.